docs/manual/installation/move-contao.de.md

```diff
@@ -77,18 +77,28 @@ Gib dein Passwort ein, wenn du danach gefragt wirst.
 {{< /tabs >}}
 
 
+## Einstellungen notieren
+Deine Einstellungen unter `System / Einstellungen` oder im Seitenroot notierst du dir am besten und trägst sie später erneut ein.
+
+{{% notice tip %}}
+Wenn du deine [Einstellungen in einer Umgebungsvariablen ablegst](https://docs.contao.org/manual/de/system/einstellungen/#umgebungsvariablen), entfällt dieser Schritt.
+{{% /notice %}} 
```

Comment:
Well, not really, you still need to copy the settings over, no? ;)

Reply:
Yes, the manual copying drops - just file copying